src/app/components/EditorInterface.tsx

```diff
@@ -142,9 +142,15 @@ const EditorInterface = ({ settings, token }: IEditorInterface) => {
       CodeBlock,
       Code,
     ],
-    content: `
-         ${settings?.content || defaultState}
-`,
+    content: settings?.content || defaultState,
+    onUpdate: ({ editor }) => {
+      if (!editor?.getHTML().includes('notification_widget')) {
+        appState?.toggleDisplayTasks({ override: false })
+      }
```

Comment:
It would be better to check whether displayTasks is already false or not. 

If displayTasks is already false, why change the state to false again, especially on each update? Checking for the value once and conditionally setting state should make it much more efficient.

```suggestion
      if (!editor?.getHTML().includes('notification_widget') && appState?.appState?.displayTasks) {
        appState?.toggleDisplayTasks({ override: false })
      }
```